wAbout/Enterprise-Login-Setup.md

```diff
@@ -0,0 +1,73 @@
+* [Auth0](https://auth0.com/docs/get-started/authentication-and-authorization-flow){:target="_blank"}{:rel="noopener noreferrer"}
+
+<blockquote class=highlight_note>
+<b>Note:</b> Users will need Interject version 2.5.0.14 or above to use federated logins. Interject can set it up where federated users and users on the existing system exist together, so we can migrate/update users individually if that makes it smoother to upgrade.
+</blockquote>
+
+The steps for setting up your Interject organization with federated logins include:
+
+* [Providing Interject OIDC Details](#provide-oidc-details)
+* [Acquiring an Enterprise Login Code](#enterprise-login-code)
+* [Whitelisting Interject domain URLs](#ip-whitelisting)
+
+### Provide OIDC Details
+
+Interject uses a set of URLs for your identity system to configure your Interject company with an identity provider for Interject to federate to. The following URLs need to be provided to the Interject development team for federated logins to be setup:
+
+* Authority URL - e.g. https://login.microsoftonline.com/{tenant}/v2.0 
+* Discovery Endpoint - e.g.  https://login.microsoftonline.com/{tenant}/v2.0/.well-known/openid-configuration  
+
+If a discovery endpoint is not available we will need the following URLs:
+
+* Authorization URL - e.g. https://login.microsoftonline.com/{tenant}/oauth2/v2.0/authorize  
+* Token URL - e.g. https://login.microsoftonline.com/{tenant}/oauth2/v2.0/token  
+* User Info URL - e.g. https://graph.microsoft.com/oidc/userinfo  
+
+<blockquote class=highlight_note>
+<b>Note:</b> For AzureAD, we are aware of a possible extra step to get federation working. Its possible your Azure AD service may need ID Tokens enabled. Learn more <a href="https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc#enable-id-tokens" target="_blank" rel="noopener noreferrer">here</a> to see if this applies to your setup.
+</blockquote>
+
+### Enterprise Login Code
+
+After the Interject team configures the identity provider for your Interject company, you will be sent an "Enterprise Login Code" that users will need to use in the Interject login page to navigate to their federated login page. See [Logging In/Out - Enterprise User](/wAbout/Logging-In-Enterprise.html) for a walkthrough on how to log in with your enterprise token and federated login.
+
+Once set up, the Enterprise Login Codes will be shown for all identity providers associated with a company in the "Organization Profile" on the [Interject portal site](https://portal.gointerject.com/).
+
+![](/images/Enterprise-Login-Setup/OrganizationProfile.png)
+<br>
+
+### IP Whitelisting
+
+If your Network uses a firewall, it might be necessary that you will need to whitelist Interject domains when users attempt to use federated logins with Interject. If this is the case, then it is important that key domains used by Interject are whitelisted on your network.
```

Comment:
i'm not sure why network is capitalized here

Reply:
I will go through it again and standardize all the caps, I get confused sometimes on what to capitalize.